backend/src/routes/v0/rag/__init__.py

```
@@ -0,0 +1,205 @@
+from fastapi.responses import JSONResponse
+from fastapi.security import HTTPBearer
+
+from src.utils.openapi import fetch_openapi_spec_sync, get_response_model
+from src.utils.retrieval import forward
+from src.constants import LANGCONNECT_SERVER_URL
+
+
+CLIENT_SPEC = fetch_openapi_spec_sync(f"{LANGCONNECT_SERVER_URL}/openapi.json") if LANGCONNECT_SERVER_URL else None
+TAG = "RAG"
+if CLIENT_SPEC:
+	gateway = APIRouter(prefix=f"/rag")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)